website/content/en/preview/upgrading/v1beta1-reference.md

```diff
@@ -604,23 +604,49 @@ In addition to these metrics, the MachineNotFound error returned by the `karpent
+| clusterCABundle                               | CLUSTER_CA_BUNDLE          | --cluster-ca-bundle      |
+| clusterName                                   | CLUSTER_NAME               | --cluster-name           |
+| clusterEndpoint                               | CLUSTER_ENDPOINT           | --cluster-endpoint       |
+| defaultInstanceProfile                        | Dropped                    | Dropped                  |
+| enablePodENI                                  | Dropped                    | Dropped                  |
+| enableENILimitedPodDensity                    | Dropped                    | Dropped                  |
+| isolatedVPC                                   | ISOLATED_VPC               | --isolated-vpc           |
+| vmMemoryOverheadPercent                       | VM_MEMORY_OVERHEAD_PERCENT | --vm-memory-overhead-percent |
+| interruptionQueueName                         | INTERRUPTION_QUEUE         | --interruption-queue     |
+| reservedENIs                                  | RESERVED_ENIS              | --reserved-enis          |
+| featureGates.driftEnabled                     | FEATURE_GATE="Drift=true"  | --feature-gates Drift=true |
+
+## Helm Values
+
+The v1beta1 helm chart comes with a number of changes to the values that were previously used in v0.31.x. Your older helm values will continue to work throughout v0.32.x but any values no longer specified in the chart will no longer be supported starting in v0.33.0.
+
+| < v0.32.x Key                           | >= v0.32.x Key                   |
+|-----------------------------------------|----------------------------------|
+| controller.outputPaths                  | logConfig.outputPaths            |
+| controller.errorOutputPaths             | logConfig.errorOutputPaths       |
+| controller.logLevel                     | logConfig.logLevel.controller    |
+| webhook.logLevel                        | logConfig.logLevel.webhook       |
+| logEncoding                             | logConfig.logEncoding            |
+| settings.aws.assumeRoleARN              | settings.assumeRoleARN           |
+| settings.aws.assumeRoleDuration         | settings.assumeRoleDuration      |
+| settings.aws.clusterCABundle            | settings.clusterCABundle         |
+| settings.aws.clusterName                | settings.clusterName             |
+| settings.aws.clusterEndpoint            | settings.clusterEndpoint         |
+| settings.aws.defaultInstanceProfile     | Dropped                          |
+| settings.aws.enablePodENI               | Dropped                          |
+| settings.aws.enableENILimitedPodDensity | Dropped                          |
+| settings.aws.isolatedVPC                | settings.isolatedVPC             |
+| settings.aws.vmMemoryOverheadPercent    | settings.vmMemoryOverheadPercent |
+| settings.aws.interruptionQueueName      | settings.interruptionQueue       |
+| settings.aws.tags                       | Dropped                          |
```

Comment:
Can we make it just super clear that this is dropped? Maybe we can bring these to the top or the bottom together? 
```suggestion
| settings.aws.defaultInstanceProfile     | **Dropped**                          |
| settings.aws.enablePodENI               | **Dropped**                          |
| settings.aws.enableENILimitedPodDensity | **Dropped**                          |
| settings.aws.isolatedVPC                | settings.isolatedVPC             |
| settings.aws.vmMemoryOverheadPercent    | settings.vmMemoryOverheadPercent |
| settings.aws.interruptionQueueName      | settings.interruptionQueue       |
| settings.aws.tags                       | **Dropped**                          |
```

Reply:
Good callout, I moved all the dropped metrics to the bottom of their tables to make it more clear